code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -336,7 +340,7 @@
 
 // look for a criminal in view of the bot
 
-/mob/living/simple_animal/bot/ed209/proc/look_for_perp()
+/mob/living/simple_animal/bot/ed209/proc/find_new_target()
 	if(disabled)
 		return
```

Comment:
You changed the other return to return FALSE, might want that here for consistency